src/plugins/intel_cpu/tests/functional/single_layer_tests/activation.cpp

```diff
@@ -154,9 +154,8 @@ std::vector<Precision> netPrc = {
         Precision::FP32
 };
 
-/* ============= Activation (1D) ============= */
```

Comment:
FYI
As I remember, `1D` means one spatial dimension - `w` (`ncw`). So previous header was correct as well 😃

Reply:
Thank you for the clarification) I will revert my changes